src/Tribe/Assets.php

```diff
@@ -518,6 +518,8 @@ public function is_style_option_full() {
 	 * @return bool
 	 */
 	public function is_settings_page() {
+		$admin_helpers = Tribe__Admin__Helpers::instance();
```

Comment:
We should not assume that a variable is set like this. It's not a class param, so define it.